guides/common/modules/proc_configuring-the-project-mcp-server.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-the-{project-context}-mcp-server"]
+= Configuring the {Project} MCP server
+
+Deploy the {Project} MCP server as a container and make it accessible to your MCP client.
+Perform these steps on the same system where your MCP client will run.
+
+.Procedure
+. Optional: Pull the latest version of the MCP container image from the registry:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ podman pull {mcp-server-image}
+----
+. Run the container, binding it to `localhost` and using the streamable HTTP transport mode:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ podman run --interactive --tty --publish localhost:8080:8080 \
```

Comment:
```suggestion
$ podman run --interactive --tty --publish 127.0.0.1:8080:8080 \
```

otherwise I'm getting
```
Error: cannot parse "localhost" as an IP address
```

This also need to be changed in the paragraph below